.clang-format

```diff
@@ -124,4 +124,5 @@ ObjCBlockIndentWidth: 4
 ### Java specific config ###
 Language:        Java
 # BreakAfterJavaFieldAnnotations: false
+JavaImportGroups: ['org.godotengine', 'android', 'com.android', 'com.google', 'java']
```

Comment:
Looks good! I'd include `androidx` after `android` to differentiate them as well.
Same for `javax` after `java`.

Reply:
Makes sense, done.